src/lucky/log_formatters/base.cr

```diff
@@ -0,0 +1,37 @@
+abstract class Lucky::LogFormatters::Base
+  abstract def format(context, time, elapsed) : String
+  
+  private def colored_status_code(status_code)
+    case status_code
+    when 200..399
+      "#{status_code.colorize(:green)}"
+    when 400..499
+      "#{status_code.colorize(:yellow)}"
+    when 500..599
+      "#{status_code.colorize(:red)}"
+    else
+      "#{status_code}"
+    end
+  end
+  
+  private def timestamp(time)
```

Comment:
Maybe this could be a public class method so it can be used by the handler as well?